services/ui_backend_service/docs/environment.md

```diff
@@ -169,4 +169,28 @@ Plaintext example:
     "end": 1618925483000
   }
 ]
-```
\ No newline at end of file
+```
+
+## Scaling reads using read replicas
+
+Databases such as [Amazon Aurora](https://aws.amazon.com/rds/aurora/) provide 
+[read replicas](https://aws.amazon.com/rds/features/read-replicas/) that make it easy to elastically scale beyond
+the capacity constraints of single database instance for heavy read workloads. You are able to separate out the reads
+and the writes of this application by setting the following two environment variables:
+
+>```
+> USE_SEPARATE_READER_POOL = 1                               
```

Comment:
I am looking for something exactly like this PR.  This variable seems a bit redundant.  It could simply be:

```
USE_SEPARATE_READER_POOL = bool(MF_METADATA_DB_READ_REPLICA_HOST)
```

thus disabling the feature if no host is given and implicitly turning it on if one is given.  This would also prevent the case where this is set to true and no host is given.